### DISCUSSION TITLE
How to DISABLE just crash events?

### DISCUSSION THREAD

**Discussion by Nerdgamerdave** [TEHE-Studios/ExpandedHelicopterEvents#153] [Q&A]
For the life of me, I can't figure out what may cause the conflict, but over the course of 3 server wipes and completely fresh restarts, I can't get the EHE crashes to work properly.

They always leave behind giant swaths of flat ground replacing the terrain with wood.  I can only assume it is a mod conflict but with so many vehicle mods and other mods on my server, I can't pinpoint it.

Is there a way to DISABLE the crashes?

That would be my preference, so we can keep the other events active.

Thank You in advance!

**Comment by Chuckleberry-Finn** [comment]
The crashes do not replace terrain - so that effect may be unrelated.
A console log may shed some light on the matter.

No, there is no sandbox option to disable crashes.

**Comment by jamesjaames** [comment]
I have the same issue. Do you know how I can fix this or which mod has caused this bug? Thank you.
